manga_translator/colorization/manga_colorization_v2.py

```diff
@@ -41,7 +43,9 @@ async def _unload(self):
     async def _infer(self, image: np.ndarray, apply_denoise=True, denoise_sigma=25) -> np.ndarray:
         # Size has to be multiple of 32
         image_width = image.shape[1]
-        size = min(image_width - image_width % 32, 1024)
+        image_height = image.shape[0]
+        extra = get_extra(image_width)
+        size = image_width + extra
```

Comment:
I shouldve told you that right away but the problem with taking the size from the image width is that the colorization model becomes painfully slow and takes up exponential amounts of vram. Thats why i capped it to 1024 and put it after the renderer.
Surely another option we have is to place it at the beginning before the upscaling but that might reduce the quality of detection.
Thats why the only choice we have is putting it after rendering, or before rendering and then resizing all the textblocks and all their properties.

Reply:
So now we have the dillema:
- Run upscaling two times (first triggered by --upscale-ratio, second to resize back after colorization)
- Or put colorization before upscaling and potentially worsen detection quality when using colorization

I feel like the second option seems a little bit better...